### DISCUSSION TITLE
start intfloat/e5-large-v2

### DISCUSSION THREAD

**Discussion by tbugfinder** [bentoml/OpenLLM#685] [Q&A]
Hello,

not sure if this is even supposed to work somehow, but is there any way to start / work with intfloat/e5-large-v2?
Somehow I'm lacking this kind of guide on pulling/ adding hf models.

Thx,

**Comment by aarnphm** [comment]
not sure I understand the question? what is this "intfloat/e5-large-v2"?

Models are pulled by default if it is a valid HF model id. It should also work with local path as long as it is a supported architecture.

**Reply by tbugfinder** [reply]
ok, I was about to look into
https://huggingface.co/intfloat/e5-large-v2

I guess I got it as the architecture is not (yet) supported.

Would it make sense to add the appropriate architecture?

**Reply by aarnphm** [reply]
This is an embedding models, which is currently out of scope for openllm.

I will close this as not planned for now
